README.md

```diff
@@ -1,6 +1,6 @@
-# Build Elm App
+# Elm AppGen
```

Comment:
I went back and forth between calling it "Elm App Gen" "Elm App Generator" and "Elm AppGen". Let me know if you prefer something different than what I picked here.  The CLI command is `elm-app-gen`, of course, but it reads oddly to always use that in the context of a document like this.

Reply:
i think "Elm App Generator" is good it's a bit longer but explicit.  and the cli name `elm-app-gen` is great for me

Reply:
done!